YJLoginSDKTests/AuthenticationRequestTests.swift

```diff
@@ -220,4 +220,50 @@ class AuthenticationRequstTests: XCTestCase {
             XCTAssertEqual(request.requestUrl, URL(string: "https://auth.login.yahoo.co.jp/yconnect/v2/authorization?bail=1&client_id=client_id&display=inapp&max_age=3600&nonce=nonce&prompt=login%20consent&redirect_uri=scheme:/&response_type=code&scope=address%20email%20openid%20profile&state=state"))
         }
     }
+
+    func test_requestUri_issuer() {
+        let optionalParameters = OptionalParameters(
+            bail: true,
+            display: .inapp,
+            maxAge: 3600,
+            prompts: [.login, .consent]
+        )
+
+        let request = AuthenticationRequest(
+            clientId: "client_id",
+            codeChallenge: "code_challenge",
+            nonce: "nonce",
+            redirectUri: URL(string: "scheme:/")!,
+            responseType: .code,
+            scopes: [.address, .email, .openid, .profile],
+            state: "state",
+            optionalParameter: optionalParameters,
+            issuer: URL(string: "https://hoge.yahoo.co.jp/yconnect/v2")!
+        )
+
+        XCTAssertNotNil(request.requestUrl)
+        let component = URLComponents(url: request.requestUrl!, resolvingAgainstBaseURL: true)
+        XCTAssertEqual(component?.scheme, "https")
+        XCTAssertEqual(component?.host, "hoge.yahoo.co.jp")
+        XCTAssertEqual(component?.path, "/yconnect/v2/authorization")
+
+        if #available(iOS 11.0, *) {
+            XCTAssertEqual(component?.percentEncodedQueryItems, [
+                URLQueryItem(name: "bail", value: "1"),
+                URLQueryItem(name: "client_id", value: "client_id"),
+                URLQueryItem(name: "code_challenge", value: "code_challenge"),
+                URLQueryItem(name: "code_challenge_method", value: "S256"),
+                URLQueryItem(name: "display", value: "inapp"),
+                URLQueryItem(name: "max_age", value: "3600"),
+                URLQueryItem(name: "nonce", value: "nonce"),
+                URLQueryItem(name: "prompt", value: "login%20consent"),
+                URLQueryItem(name: "redirect_uri", value: "scheme:/"),
+                URLQueryItem(name: "response_type", value: "code"),
+                URLQueryItem(name: "scope", value: "address%20email%20openid%20profile"),
+                URLQueryItem(name: "state", value: "state"),
+            ])
+        } else {
+            XCTAssertEqual(request.requestUrl, URL(string: "https://auth.login.yahoo.co.jp/yconnect/v2/authorization?bail=1&client_id=client_id&display=inapp&max_age=3600&nonce=nonce&prompt=login%20consent&redirect_uri=scheme:/&response_type=code&scope=address%20email%20openid%20profile&state=state"))
```

Comment:
@kazuki229 I think the host of `request.requestUrl` should be assert `hoge.yahoo.co.jp` but `auth.login.yahoo.co.jp`.

Reply:
@tona0516 
Fixed https://github.com/yahoojapan/yjlogin-ios-sdk/pull/13/commits/27491463d3316a17f684a8040a61a9cb11a366a8